### PR TITLE
Scaffold milestone2

### PR DESCRIPTION
**Pull request by prolajumokeoni**
## Milestone 2
In this milestone created:
- Created a friendship model with associations 
- Users can see other posts from his friend
- Users can send, reject or accept a friend request
### Previous changes requested by TSE have been implemented

### CODE REVIEW

File: app/assets/javascripts/friendships.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (bellom) [inline app/assets/javascripts/friendships.coffee]: Let's be professional with our codebase, kindly delete this empty file.

File: app/assets/stylesheets/friendships.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the friendships controller here.
```
Comment (bellom) [inline app/assets/stylesheets/friendships.scss]: kindly delete this empty file.

File: app/controllers/friendships_controller.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+class FriendshipsController < ApplicationController
```
Comment (bellom) [inline app/controllers/friendships_controller.rb:1]: kindly delete this empty file.

File: app/helpers/friendships_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module FriendshipsHelper
```
Comment (bellom) [inline app/helpers/friendships_helper.rb]: kindly delete this empty file.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @prolajumokeoni  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

1. Let's a button to invite users to friendship on the users list page and on the single-user page ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/user_can_see_invite_buttton.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))


2. Let's make sure posts timeline includes posts written by logged-in user and their friends ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/friends_timeline.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))

3. Let's add unit and Integration tests! ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/tests.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))

4. Kindly check out my comments below;


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

File: app/assets/javascripts/friendships.coffee (hunk 1 above)
Comment (carlos-ssh) [reply app/assets/javascripts/friendships.coffee]: - Kindly cover the requirements from your last reviewer.

File: app/assets/stylesheets/friendships.scss (hunk 2 above)
Comment (carlos-ssh) [reply app/assets/stylesheets/friendships.scss]: - Kindly cover the requirements from your last reviewer.

File: app/assets/javascripts/friendships.coffee (hunk 1 above)
Comment (carlos-ssh) [reply app/assets/javascripts/friendships.coffee]: - Kindly cover the requirements from your last reviewer.

File: app/assets/stylesheets/friendships.scss (hunk 2 above)
Comment (carlos-ssh) [reply app/assets/stylesheets/friendships.scss]: - Kindly cover the requirements from your last reviewer.

File: app/helpers/friendships_helper.rb (hunk 4 above)
Comment (carlos-ssh) [reply app/helpers/friendships_helper.rb]: - Kindly cover the requirements from your last reviewer.

File: app/assets/javascripts/friendships.coffee [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (carlos-ssh) [inline app/assets/javascripts/friendships.coffee]: - Kindly add FK for your post user_id into your erd.png.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @prolajumokeoni , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @prolajumokeoni , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**